scene/resources/height_map_shape_3d.cpp

```diff
@@ -187,6 +188,48 @@ real_t HeightMapShape3D::get_max_height() const {
 	return max_height;
 }
 
+void HeightMapShape3D::update_map_data_from_image(const Ref<Image> &p_image, real_t p_height_min, real_t p_height_max) {
+	ERR_FAIL_COND(p_image.is_null());
+	ERR_FAIL_COND(p_image->get_format() != Image::FORMAT_RF);
+	ERR_FAIL_COND(p_image->get_width() < 2);
+	ERR_FAIL_COND(p_image->get_height() < 2);
+	ERR_FAIL_COND(p_height_min > p_height_max);
```

Comment:
Missed earlier but these should likely be changed to `ERR_FAIL_COND_MSG` with user-friendly messages (especially given that e.g. it's not mentioned in the docs that image needs to be at least 2x2 etc.).

Reply:
Agree, I will update the pr with error macros msgs for all the setters. The parameter names are also not in sync.